modules/ipi-install-configuring-nodes.adoc

```diff
@@ -87,3 +87,33 @@ To enable Secure Boot manually, refer to the hardware guide for the node and exe
 ====
 Red Hat does not support Secure Boot with self-generated keys.
 ====
+
+[discrete]
+== Configuring the Compatibility Support Module for Fujitsu iRMC
```

Comment:
This suggestion might be out of context for this PR, but you could consider making each of these procedure sections individual module files.

You could also add an ID for this heading in a line under the `[discrete]` reference, so that the section can be directly cross-referenced.